src/core/dbus.c

```diff
@@ -977,7 +977,7 @@ static int bus_init_private(Manager *m) {
                 char *p = sa.un.sun_path;
                 const char *e;
 
-                e = secure_getenv("XDG_RUNTIME_DIR");
+                e = getenv("XDG_RUNTIME_DIR");
```

Comment:
I seriously doubt this really helps? We use the expression ` secure_getenv("XDG_RUNTIME_DIR")` at various places, including in sd-bus, hence the above doesn't really bring any benefits?